### ISSUE TITLE
We need different AIA images for the test data, or we need to fix the scaling.

### ISSUE THREAD

**Issue by wafels** [sunpy/sunpy#2726]
This is because the ones we have right now make our default scaling look terrible.  It's a terrible advert for SunPy.

**Comment by Cadair** [comment]
You mean sample data and not test data right? The choice of sample data was @ehsteve and Andy iirc

**Comment by Cadair** [comment]
My preferred solution to this is to make our default scaling smarter.

**Comment by wafels** [comment]
Yes the sample data (I edited the title of the issue). The problem is the sample data we use contains a small number of pixels that have high values and those high values are not common across the AIA data. Therefore the default scaling does not work.  Clipping the image range would work, or a stretch function that accommodates these higher values.  Both may need to be AIA channel dependent.

**Comment by ehsteve** [comment]
I thought we already had an issue to come up with a smarter set of sample data that had a "theme" to it like perhaps a particular flare.

**Comment by Cadair** [comment]
We did that already.

**Comment by ehsteve** [comment]
Ah right. Well we did a crappy job then!

**Comment by ehsteve** [comment]
The correct way to do this would be to mask the high pixels which may be bad or cosmic ray hits. I'm not sure that we want to come up with a smarter color table to deal with this as it would be to the detriment of the good data. I would vote to find images slightly before or after the ones we have here that hopefully do not have those bad pixels.

**Comment by wafels** [comment]
Yes, masking is one way to fix this.  My preference is to use some new data.  What the current selection based on the RHESSI or GOES flare peak?  My bet is that shortly after time the AIA pixels return to normal values but there is still some post-flare emission observable in RHESSI.

**Comment by Cadair** [comment]
We will need to add a v2 folder to the sample data repo: https://github.com/sunpy/sample-data

**Comment by wtbarnes** [comment]
To be clear, is the primary issue when you plot our sample data, a lot of it looks like this?

<img width="594" height="479" alt="Image" src="https://github.com/user-attachments/assets/17281941-ca26-4a99-bf85-f33519b33cf8" />

FWIW, we have had users confused in the past that there AIA images were "dark". I'm not really sure of the right approach to take to find a "better" set of sample data.

**Comment by Cadair** [comment]
We picked the current set of sample data so that it was all showing an interesting event (well as much of it as possible), and now people complain there's a flare in it!
